docker-compose.yml

```diff
@@ -14,9 +14,10 @@ services:
   api:
     healthcheck:
       test: test ! -e /sdr_unhealthy && test ! -e /scheduler_dead
-      interval: 10s
+      interval: 6s
       timeout: 3s
       retries: 1
+    shm_size: '16gb'
```

Comment:
This shared memory configuration may not be best to include all the time- this is chosen specifically for SEA NUCs with 32GB total RAM. Only the SEA action makes use of shared memory at all.

Reply:
I added a section to the README about this, per Doug's suggestion.